enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/catchup/storecopy/StoreResource.java

```diff
@@ -51,19 +52,19 @@
 
     ReadableByteChannel open() throws IOException
     {
-        Optional<PagedFile> existingMapping = pageCache.getExistingMapping( file );
-        if ( existingMapping.isPresent() )
+        boolean isDefaultFileSystem = pageCache.getCachedFileSystem() instanceof DefaultFileSystemAbstraction;
```

Comment:
can we move this knowledge into one place inside a page cache? there is no reason why all of the places should be aware of this implementation detail?

Reply:
Good idea. Will try to come up with a proper helper method for this.